packit-ci.fmf

```diff
@@ -12,7 +12,7 @@ context:
 prepare:
  - how: shell
    script:
-    - ln -s $(pwd) /var/tmp/keylime_sources
+    - test -e /var/tmp/keylime_sources || ln -s $(pwd) /var/tmp/keylime_sources
```

Comment:
FYI: typo in patch description: additonal

Reply:
fixed